Makefile

```diff
@@ -45,6 +46,11 @@ INTEGRATION_TEST_ARGS += -eksctl.version=$(INTEGRATION_TEST_VERSION)
 $(info will launch integration tests for Kubernetes version $(INTEGRATION_TEST_VERSION))
 endif
 
+.PHONY: github-build
+github-build:
+	cp -a . $(GOPATH)/src/$(import_prefix)
```

Comment:
@2opremio I believe we won't have to do this, if we switch to modules...

Reply:
Exactly, we won't :)